crates/cardano/src/model.rs

```diff
@@ -533,6 +533,9 @@ pub struct AccountState {
 
     #[n(6)]
     pub credential: StakeCredential,
+
+    #[n(7)]
+    pub retired_pool: Option<PoolHash>,
```

Comment:
let's put a cbor default attribute here so that previous snapshots still work regardless of the existence of the new value.